src/main/resources/config/application-dev.yml

```diff
@@ -163,9 +163,16 @@ application:
   frontend:
     sentry-project-id:
     google-analytics-project-id:
+    recaptcha-project-id:
+    recaptcha-site-key:
+    recaptcha-threshold:
   aws:
     s3-access-key:
     s3-secret-key:
     s3-region:
   github-token:
   db-read-only: false # Certain endpoints will be disabled when readonly is set to true
+  recaptcha:
+    recaptcha-project-id:
+    recaptcha-site-key:
+    recaptcha-threshold:
```

Comment:
Can we get rid of the prefix? We don't need the recaptcha if they already under the recaptcha property

Reply:
Yes I can get rid of that